subprojects/module-test/module-test.gradle

```diff
@@ -1,5 +1,5 @@
-plugins {
-    id 'java'
+if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
```

Comment:
Please use `>`. `!=` is somehow confusing for readers. `JavaVersion` is `Comparable`, so it will work.

Reply:
The last commit changes to `==`, as it had to also change to a positive in this case. This check is only for Travis until we remove JDK8 from our matrix.